src/tjota_db.erl

```diff
@@ -84,14 +85,63 @@ create_table(user) ->
         )
     ", [?KEYSPACE, ?TABLE_USER]));
 
-create_table(session) -> not_implemented;
-create_table(room) -> not_implemented;
-create_table(message) -> not_implemented;
+create_table(session) -> 
+    {ok, Client} = get_cqerl_client(),
+    {ok, _} = cqerl:run_query(Client, io_lib:format("
+        CREATE TABLE IF NOT EXISTS ~s.~s (
+        )
+    ", [?KEYSPACE, ?TABLE_SESSION]));
+
 
-create_table(user_room) -> not_implemented;
-create_table(room_user) -> not_implemented.
+create_table(room) -> 
+    {ok, Client} = get_cqerl_client(),
+    {ok, _} = cqerl:run_query(Client, io_lib:format("
+        CREATE TABLE IF NOT EXISTS ~s.~s (
+            room-name TEXT,
```

Comment:
`room-name` will make the query crash. In CQL as well as SQL `-` is minus so names cannot contain `-`. However, they may contain `_` (underscore).

The output would be similar to this.
```
SyntaxException: <ErrorMessage code=2000 [Syntax error in CQL query] message="line 1:18 no viable alternative at input '-' (CREATE TABLE ... (room[-]...)">
```